carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carma_carla_guidance

```diff
@@ -35,15 +35,14 @@ def initialize():
     rospy.init_node("carma_carla_guidance")
     service = rospy.ServiceProxy('/set_active_guidance')
```

Comment:
Could you double check with this line of code a little bit? I am afraid that is not a acceptable usage. This is the usage I found from ROS python official website: [link](http://wiki.ros.org/ROS/Tutorials/WritingServiceClient%28python%29#:~:text=11%20%20%20%20%20try%3A%0A%20%2012-,add_two_ints%20%3D%20rospy.ServiceProxy(%27add_two_ints%27%2C%20AddTwoInts),-13%20%20%20%20%20%20%20%20%20resp1%20%3D%20add_two_ints)